docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -375,6 +409,14 @@ const buildComponentDocumentation = async (options: {
 
       const deprecation = (propDescriptor.description || '').match(/@deprecated(\s+(?<info>.*))?/);
 
+      let signature;
```

Comment:
If we start reusing `ReactApi` interface from the Core, we can use it here as well:

```suggestion
      let signature: CoreReactApi['propsTable']['string']['signature'];
```